packages/sample-v2-addon/package.json

```diff
@@ -64,17 +51,14 @@
     "@babel/core": "^7.21.4",
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/plugin-proposal-decorators": "^7.21.0",
-    "@babel/preset-typescript": "^7.21.4",
-    "@babel/runtime": "^7.21.5",
```

Comment:
In https://github.com/embroider-build/addon-blueprint/pull/113, Nick wondered:

> I am not certain if it is also needed for JS addons, since that uses a different rollup plugin which has a "bundle helpers" specified by default already in [the rollup config](https://github.com/embroider-build/addon-blueprint/blob/main/files/__addonLocation__/rollup.config.mjs#L43).

It seems that, for JS projects, we don't need to add `@babel/runtime` to `devDependencies`. After updating `sample-v2-addon` to be a JS project, I could run `docs-app` without encountering the `@babel/runtime`-error.